conf/proxy.conf

```diff
@@ -285,3 +285,9 @@ zookeeperSessionTimeoutMs=-1
 # ZooKeeper cache expiry time in seconds
 # Deprecated: use metadataStoreCacheExpirySeconds
 zooKeeperCacheExpirySeconds=-1
+
+
+### --- Metrics
```

Comment:
Please move this part before `Deprecated config variables`

Reply:
Resolved